styles/list.css

```diff
@@ -0,0 +1,19 @@
+@import url(/styles/index.css);
+
+.list-normal {
+  text-decoration: none;
+  line-height: 2rem;
+}
+
+.stacked-list li {
+  font-size: 1.6rem;
```

Comment:
You can use variables instead of absolute values

Reply:
Thanks for bring that to notice, i have used utility class instead of absolute font-size